src/components/ReactSearchAutocomplete.js

```diff
@@ -64,7 +67,7 @@ export default function ReactSearchAutocomplete(props) {
   }, [inputSearchString])
 
   useEffect(() => {
-    searchString?.length > 0 && results?.length > 0 && setResults(fuseResults(searchString))
+    searchString?.length > 0 && results?.length > 0 && setResults(fuseResults(searchString)) && setKeyLocation(null)
```

Comment:
@harveytoro If the `items` change I think you want to reset the selected item (with keyboard) regardless if `searchString?.length > 0 && results?.length > 0` is `true`, therefore I'd just add `setKeyLocation(null)` on a separate line.